contracts/Inbox.sol

```diff
@@ -55,7 +55,7 @@ contract Inbox is IInbox, Ownable {
         }
     }
 
-    function version() external pure returns (string memory) { return "v0.0.3-beta"; }
+    function version() external pure returns (string memory) { return "v0.0.4211-beta"; }
```

Comment:
for simplicity i think we should create a version contract that is inherited by evertything else - that way we only have to update one contract, and it doesnt muddy up the code. wont affect this approval but yeah

Reply:
May be a version library i.e. something like [this](https://github.com/the-eco-foundation/eco-routes/blob/ED-4357/contracts/libs/Semver.sol) Already done in BETA.
```
// SPDX-License-Identifier: UNLICENSED
pragma solidity ^0.8.28;

library Semver {
    function version() internal pure returns (string memory) {
        return "0.3.1-beta.0";
    }
}
```

Reply:
ill copy it